fhir-install/src/main/docker/ibm-fhir-term-graph-loader/Dockerfile

```diff
@@ -4,25 +4,62 @@
 # SPDX-License-Identifier: Apache-2.0
 # ----------------------------------------------------------------------------
 
-# Build stage
-FROM maven:3.6.0-jdk-11-slim AS build
-COPY . /
+# Stage: Base
 
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -f /fhir-examples/pom.xml install -DskipTests=true
+# IBM Semeru Runtimes provides Non-official docker images as part of this repo. These are maintained by IBM.
+# The link to Semeru is at https://hub.docker.com/r/ibmsemeruruntime/open-11-jdk
+FROM ibmsemeruruntime/open-11-jdk:ubi_min-jre-latest as base
 
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -f /fhir-parent/pom.xml install -DskipTests=true -pl ../fhir-term-graph -am
+# Create the base working directory
+RUN mkdir -p /opt/ibm/fhir/graph/loader/workarea
 
-RUN mvn org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy-dependencies -f /fhir-parent -pl ../fhir-term-graph -am -DoutputDirectory=target/ftg
+# Copy in the relevant artifacts in a single command
+COPY ./run.sh ./target/fhir-term-graph-loader-*-cli.jar ./target/LICENSE ./target/ftg/ /opt/ibm/fhir/graph/loader/
 
-# Package stage
-FROM openjdk:11-jre-slim
+# ----------------------------------------------------------------------------
+# Stage: Runnable
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+ARG FHIR_VERSION=4.10.0
```

Comment:
How do we keep this up-to-date with the build?

Reply:
you don't the build takes care of inserting it.